frontend/src/api.ts

```diff
@@ -320,8 +320,8 @@ export const createInvite = authorized<
  * @param onSuccess Fired on successful request.
  * @param onFailure Fired on failed request.
  */
-const dropStudent = authorized<{ classId: string; studentId: string }, void>(
-  ({ classId, studentId }) => [`/class/${classId}/drop`, { studentId }]
+export const dropStudent = authorized<{ classId: string; id: string }, void>(
+  ({ classId, id }) => [`/class/${classId}/drop`, { id }]
 );
```

Comment:
This isn't very idiomatic, we can keep the meaning in the function signature without breaking the code.

```suggestion
export const dropStudent = authorized<{ classId: string; studentId: string }, void>(
  ({ classId, id }) => [`/class/${classId}/drop`, { id: studentId }]
);
```